compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -112,14 +112,14 @@ int entry(int argc, char **argv)
   add_switch(arser, "--fuse_mean_with_mean",
              "This will fuse two Mean operations when they follow one by one. This will fold them "
              "into one operation and merge reduction indices.");
-  add_switch(arser, "--fuse_mul_with_fully_connected",
-             "This will fuse Mul operator to FullyConnected operator");
   add_switch(arser, "--fuse_mul_to_fullyconnected_weights",
              "This will fuse Mul to following FullyConnected weights");
   add_switch(arser, "--fuse_mul_with_conv",
              "This will fuse Mul operation with a preceding Conv if possible.");
   add_switch(arser, "--fuse_mul_with_div",
              "This will fuse Mul operation with a Div operation whose numerator is const.");
+  add_switch(arser, "--fuse_mul_with_fully_connected",
```

Comment:
```suggestion
  add_switch(arser, "--fuse_mul_with_fullyconnected",
```
to follow other options

Reply:
Got it! Done in scope of https://github.com/Samsung/ONE/pull/13528/commits/d38654162c2aa13c4b1679da714a72ab4d3e8578